docs/content/reference/infrastructure/aws.md

```diff
@@ -0,0 +1,86 @@
+
+## Infrastructure - AWS
+
+In this section we want to dissect who creates what and what not. It contains 4 stages:
+
+- Infra pre-required and unmanaged for hypershift operator in an arbitrary AWS account
+- Infra pre-required and unmanaged in Hosted Cluster AWS account
+- Infra managed by hypershift in Management AWS account
+- Infra managed by hypershift in Hosted Cluster AWS account
+- Infra managed by kubernetes in Hosted Cluster AWS account
+
+!!! note
+    The arbitrary AWS account depends on who is providing the Hypershift service.
+
+    - **Self Managed:** It will be controlled by the [Cluster Service Provider](../concepts-and-personas.md#personas).
+    - **SaaS:** In this case the AWS Account will belong to Red Hat.
+
+#### Infra pre-required and unmanaged for hypershift operator in an arbitrary AWS account
+
+=== "Management Cluster AWS Account"
+    - 1 S3 Bucket
+        - OIDC
+        - OLM Registry for Private Hosted Cluster
+    - Route 53 Hosted zones
+        - Domain to host Private and Public entries for HostedClusters
+
+#### Infra pre-required and unmanaged in Hosted Cluster AWS account
+
+=== "Public"
+    - 1 VPC
+    - 1 DHCP Options
+    - 2 Subnets
+        - 1 Public Subnet (for Public HostedCluster)
+        - 1 Private Subnet (Access from the Instances, Dataplane)
+    - 1 Internet Gateway
+    - 1 Elastic IP
+    - 1 NAT Gateway
+    - 1 Security Group (Worker Nodes)
+    - 1 Route Tables (Public)
+    - 1 Ingress Service Load Balancer
+
+=== "Private"
+    - 1 VPC
+    - 1 DHCP Options
+    - 1 Private Subnet (Access from the Instances, Dataplane)
```

Comment:
Private subnet - internal data plane subnet
Public subnet - enable access to the internet from the data plane

Reply:
HC are public/private in the sense of the hosted cluster endpoints. The data plane still needs to have access to the internet. Otherwise would be a disconnected cluster which we are not getting into here.